test/otherFactory.js

```diff
@@ -17,19 +17,19 @@
                 });
             }
         },
-        WordCloud: {
-            simple: function (callback) {
-                require(["test/DataFactory", "src/other/WordCloud"], function (DataFactory, WordCloud) {
-                    var words = DataFactory.WordCloud.simple.words.map(function (d) {
-                        return [d, 10 + Math.random() * 14];
-                    });
-                    callback(new WordCloud()
-                        .columns(DataFactory.WordCloud.simple.columns)
-                        .data(words)
-                    );
-                });
-            }
-        },
+        // WordCloud: {
```

Comment:
how come word cloud is removed .. if its because its not working ... i have that fixed in several PRs that are now passing travis ...